torch/onnx/symbolic_opset9.py

```diff
@@ -2911,9 +2911,12 @@ def baddbmm(g, self, batch1, batch2, beta, alpha):
 def meshgrid(g, tensor_list, indexing: Optional[str] = None):
     if indexing is None:
         indexing = 'ij'
-    elif indexing != 'ij':
+    elif indexing not in {'ij', 'xy'}:
```

Comment:
Another option would be to throw an error when trying to export meshgrid with 'xy' indexing and file an issue (linked to in a comment) for our ONNX partners to follow-up

Reply:
good to know, but that ship has sailed. thankfully this was easy to fixup and it was a good lesson.